src/main/resources/static/widget.js

```diff
@@ -0,0 +1,36 @@
+function getIframe() {
+  let tcTokenURL = new URL(document.currentScript.src).searchParams.get(
+    "tcTokenURL"
+  );
+  if (tcTokenURL === null) {
+    tcTokenURL = document.getElementById("useid-widget").dataset.tcTokenUrl;
+
+    if (tcTokenURL === undefined) {
+      const error = document.createElement("div");
+      error.innerHTML = "Fehlerhafte Konfiguration: TC Token nicht definiert.";
+      return error;
+    }
+  }
+
+  const iframe = document.createElement("iframe");
+  iframe.setAttribute(
+    "src",
+    `https://useid.dev.ds4g.net/widget?tcTokenURL=${tcTokenURL}`
+  );
+  iframe.style.border = "2px solid #B8BDC3";
+  iframe.style.borderRadius = "20px";
+  iframe.style.width = "100%";
+  iframe.style.minHeight = "600px"; // TODO: Set height to fit
```

Comment:
I would leave styling to the surrounding container. Just use width and height 100%. Shouldn't that be enough?

Reply:
Yes, agree. It was for now to have some nicer look until the design is done. Unfortunately, handling the height is not that easy, because the parent can't access the height of the iframe. So I think we should at least handle that, to show the full widget.

Reply:
Understand. Re height: Can we calculate the needed height according to the content of the iframe?